CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# dbt_fivetran_utils v0.4.2
```

Comment:
0.4.2 hasn't been released yet. We should fold these into the 0.4.1 changelog entries instead.

Reply:
updated